### ISSUE TITLE
Set preset mode to none

### ISSUE THREAD

**Issue by phdelodder** [zxdavb/evohome_cc#14]
```
2021-03-07 13:48:38 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [140170447772976] isinstance() arg 2 must be a type or tuple of types
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 137, in handle_call_service
    await hass.services.async_call(
  File "/usr/src/homeassistant/homeassistant/core.py", line 1488, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1523, in _execute_service
    await handler.job.target(service_call)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 204, in handle_service
    await self.hass.helpers.service.entity_service_call(
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 642, in entity_service_call
    future.result()  # pop exception if have
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 681, in async_request_call
    await coro
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 679, in _handle_entity_call
    await result
  File "/usr/src/homeassistant/homeassistant/components/climate/__init__.py", line 446, in async_set_preset_mode
    await self.hass.async_add_executor_job(self.set_preset_mode, preset_mode)
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/evohome_cc/climate.py", line 255, in set_preset_mode
    self._evo_device.reset_mode()
  File "/usr/local/lib/python3.8/site-packages/evohome_rf/zones.py", line 824, in reset_mode
    return self.set_mode(mode="follow_schedule")
  File "/usr/local/lib/python3.8/site-packages/evohome_rf/zones.py", line 815, in set_mode
    cmd = Command.set_zone_mode(self._ctl.id, self.idx, mode, setpoint, until)
  File "/usr/local/lib/python3.8/site-packages/evohome_rf/command.py", line 512, in set_zone_mode
    elif not isinstance(setpoint, (None, float)):
TypeError: isinstance() arg 2 must be a type or tuple of types
```

**Comment by phdelodder** [comment]
```
2021-03-07 14:11:53 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [140526827342144] follow_schedule
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 137, in handle_call_service
    await hass.services.async_call(
  File "/usr/src/homeassistant/homeassistant/core.py", line 1488, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1523, in _execute_service
    await handler.job.target(service_call)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 204, in handle_service
    await self.hass.helpers.service.entity_service_call(
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 642, in entity_service_call
    future.result()  # pop exception if have
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 681, in async_request_call
    await coro
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 679, in _handle_entity_call
    await result
  File "/usr/src/homeassistant/homeassistant/components/climate/__init__.py", line 446, in async_set_preset_mode
    await self.hass.async_add_executor_job(self.set_preset_mode, preset_mode)
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/evohome_cc/climate.py", line 255, in set_preset_mode
    self._evo_device.reset_mode()
  File "/usr/local/lib/python3.8/site-packages/evohome_rf/zones.py", line 824, in reset_mode
    return self.set_mode(mode="follow_schedule")
  File "/usr/local/lib/python3.8/site-packages/evohome_rf/zones.py", line 815, in set_mode
    cmd = Command.set_zone_mode(self._ctl.id, self.idx, mode, setpoint, until)
  File "/usr/local/lib/python3.8/site-packages/evohome_rf/command.py", line 520, in set_zone_mode
    assert mode in ZONE_MODE_MAP, mode
AssertionError: follow_schedule
```
After updating to 0.5.19

**Comment by phdelodder** [comment]
Fixed in 0.5.20
